ingestion_tools/scripts/importers/annotation.py

```diff
@@ -85,12 +85,12 @@ def __init__(
         glob_string: str,
         glob_vars: dict[str, str],
         file_format: str,
-        is_viz_default: bool = False,
+        is_visualization_default: bool = False,
     ):
         self.glob_string = glob_string.format(**glob_vars)
         self.file_format = file_format
         self.shape = shape
-        self.is_viz_default = is_viz_default
+        self.is_viz_default = is_visualization_default
```

Comment:
Can we also rename the class variable to `is_visualization_default` for consistency?

Reply:
done with the last commit.